### ISSUE TITLE
commandButton onclick

### ISSUE THREAD

**Issue by chongma** [TheCoder4eu/BootsFaces-OSP#893]
`b:commandButton` `onclick` method needs to be protected from returning `true`.  e.g. 

```xml
<b:commandButton value="Test" action="#{inputTextBean.update}"
	onclick="return confirm('Are you sure?')">
	<f:ajax render="@form" />
</b:commandButton>
```
results in a page reload.

```xml
<b:commandButton value="Test" action="#{inputTextBean.update}"
	onclick="if (! confirm('Are you sure?')) return false">
	<f:ajax render="@form" />
</b:commandButton>
```
solves the problem.

please see https://stackoverflow.com/questions/4148689/jsf-how-do-i-implement-a-javascript-are-you-sure-prompt-for-a-hcommandbut/4148853?noredirect=1#comment82101529_4148853 for discussion.

also i have written a reproducer on https://github.com/chongma/test called commandButtonOnClick

**Comment by stephanrauh** [comment]
What's happening here is that the `return false` suppresses the AJAX call (which is the desired effect) and replaces it with a simple form submit (which isn't desired at all).

**Comment by stephanrauh** [comment]
@chongma  I've solved the bug now. Can you double-check it, please?

**Comment by chongma** [comment]
has this been released now?  i will test it with 1.2.0.  also what is the version of the new snapshot for latest version?  2.0-SNAPSHOT?

**Comment by stephanrauh** [comment]
Yes, we've released this feature as part of BootsFaces 1.2.0.

The most current snapshot is going to be 1.2.1. Currently, there's none, because we haven't done anything in the 1.2 branch since the release. As for the 2.0 version, I guess we'll move to a public repository when we've managed to implement 20 components. Currently, we're still busy laying the foundations, developing an efficient development process and so on.
